aspnetcore/tutorials/first-mvc-app/new-field.md

```diff
@@ -58,13 +58,13 @@ Edit the */Views/Movies/Index.cshtml* file and add a `Rating` field:
 
 ::: moniker range=">= aspnetcore-3.0 < aspnetcore-5.0"
 
-[!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Views/Movies/IndexGenreRating.cshtml?highlight=16,38&range=24-63)]
+[!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/SnapshotNewField3/Views/Movies/IndexGenreRating.cshtml?highlight=16,38&range=24-63)]
```

Comment:
> The 3.x and 5 samples represent an earlier state of the application in the series and will not compile after download with IndexGenreRating.cshtml example in them.

That's because https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Models/Movie.cs is defined. That's the wrong file. It should be  https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Models/MovieDateRatingDAmult.cs

The right way to fix this is define the correct movie model for the completed applications.